bad_connector_woocommerce/tests/test_woo_backend.py

```diff
@@ -109,3 +310,40 @@ def test_backend_cron(self):
         self.backend.cron_export_sale_order_status()
         self.backend.cron_update_stock_qty()
         self.backend.cron_import_product_templates()
+        self.backend.generate_token()
+
+    def test_product_create_webhook(self):
+        """Called webhook for Product"""
+        product_webhook_url = "/update_product/woo_webhook/{}".format(
+            self.backend.access_token
```

Comment:
I think here, we have to pass the self.backend.test_acess_token, as the backend currently is in test mode, if it is, then do it all places.